_opensearch/segment-replication/configuration

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Segment Replication Configuration
+nav_order: 122
+parent: Segment Replication Feature
+grand_parent: Opensearch
+---
+
+## Segment Replication Configuration
+
+### Enabling the feature flag
```

Comment:
Same here - a descriptive sentence that separates the headings.

Reply:
kk